.readthedocs.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+formats: all
+sphinx:
+  configuration: docs/conf.py
+  fail_on_warning: true
+python:
+  install:
+    requirements: requirements.txt
```

Comment:
We need to change this later, when https://github.com/rtfd/readthedocs.org/pull/4740 is deployed

Reply:
I wonder if we should try and deploy this first.

Reply:
Cool, lets ship it then :)